src/fpm_filesystem.f90

```diff
@@ -344,8 +343,7 @@ recursive subroutine list_files(dir, files, recurse)
     end select
 
     if (stat /= 0) then
-        print *, 'execute_command_line() failed'
-        error stop
+        stop 'execute_command_line() failed'
```

Comment:
This stops with exit status 0

Reply:
Oops. I thought "error stop" always exited non-zero; but I use my own routine that does a message and a number.